lowcodelanguage/src/main/java/net/minecraftforge/fml/lowcodemod/LowCodeModContainer.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (c) Forge Development LLC and contributors
+ * SPDX-License-Identifier: LGPL-2.1-only
+ */
+
+package net.minecraftforge.fml.lowcodemod;
+
+import net.minecraftforge.eventbus.api.Event;
+import net.minecraftforge.fml.ModContainer;
+import net.minecraftforge.fml.event.IModBusEvent;
+import net.minecraftforge.forgespi.language.IModInfo;
+import net.minecraftforge.forgespi.language.ModFileScanData;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+
+import static net.minecraftforge.fml.loading.LogMarkers.LOADING;
+
+public class LowCodeModContainer extends ModContainer
+{
+    private static final Logger LOGGER = LogManager.getLogger();
+    private final ModFileScanData scanResults;
+    private Object modInstance;
+
+    public LowCodeModContainer(IModInfo info, ModFileScanData modFileScanResults, ModuleLayer gameLayer)
+    {
+        super(info);
+        LOGGER.debug(LOADING,"Creating LowCodeModContainer for {}", info.getModId());
```

Comment:
This would likely be better to use SLF4J but 1.18.2 will never be switching off log4j so meh.

Reply:
slf4j would add significant complexity to the project, FYI.